tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/Stylesheet/Stylesheet.spec.ts

```diff
@@ -17,7 +17,7 @@ test.describe('Stylesheet tests', () => {
 
   test('can create a stylesheet', async ({umbracoApi}) => {
     // Act
-    stylesheetPath = await umbracoApi.stylesheet.create(stylesheetName, 'content');
+    await umbracoApi.stylesheet.create(stylesheetName, 'content');
```

Comment:
Since we don't update the value for stylesheetPath, would the value not be empty when used in the tests?